runtime error: makeslice: len out of range

When following the instructions in the [contrib/cluster-maxscale/README.md](/mariadb-corporation/replication-manager/blob/0.7/contrib/cluster-maxscale/README.md) file I get this error after all the other services have been started.
~~~
panic: runtime error: makeslice: len out of range
replication-manager_1  | 
replication-manager_1  | goroutine 1 [running]:
replication-manager_1  | panic(0x972cc0, 0xc820278d20)
replication-manager_1  | 	/usr/local/go/src/runtime/panic.go:481 +0x3e6
replication-manager_1  | main.glob.func10(0xcf9460, 0xc820084c80, 0x0, 0x5)
replication-manager_1  | 	/go/src/replication-manager/repmgr.go:220 +0x35e
replication-manager_1  | replication-manager/vendor/github.com/spf13/cobra.(*Command).execute(0xcf9460, 0xc820084b40, 0x5, 0x5, 0x0, 0x0)
replication-manager_1  | 	/go/src/replication-manager/vendor/github.com/spf13/cobra/command.go:565 +0x85a
replication-manager_1  | replication-manager/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0xcf8860, 0xcf9460, 0x0, 0x0)
replication-manager_1  | 	/go/src/replication-manager/vendor/github.com/spf13/cobra/command.go:651 +0x55c
replication-manager_1  | replication-manager/vendor/github.com/spf13/cobra.(*Command).Execute(0xcf8860, 0x0, 0x0)
replication-manager_1  | 	/go/src/replication-manager/vendor/github.com/spf13/cobra/command.go:610 +0x2d
replication-manager_1  | main.main()
replication-manager_1  | 	/go/src/replication-manager/main.go:30 +0x27
~~~

~~~
$ docker-compose ps                                                                                                                                                                                         Name                               Command               State             Ports          
---------------------------------------------------------------------------------------------------------
clustermaxscale_agent1_1                replication-manager agent  ...   Up                               
clustermaxscale_agent2_1                replication-manager agent  ...   Up                               
clustermaxscale_agent3_1                replication-manager agent  ...   Up                               
clustermaxscale_mariadb1_1              docker-entrypoint.sh --log ...   Up       3306/tcp                
clustermaxscale_mariadb2_1              docker-entrypoint.sh --log ...   Up       3306/tcp                
clustermaxscale_mariadb3_1              docker-entrypoint.sh --log ...   Up       3306/tcp                
clustermaxscale_maxscale_1              /usr/bin/maxscale --nodaem ...   Up       0.0.0.0:32772->3306/tcp 
clustermaxscale_replication-manager_1   replication-manager monito ...   Exit 2     
~~~

Passing `--daemon=true` in the command serves as workarround.

--- a/contrib/cluster-maxscale/docker-compose.yml
+++ b/contrib/cluster-maxscale/docker-compose.yml
@@ -18,6 +18,7 @@ services:
       - agent2
       - agent3
     tty: true
+    stdin_open: true
 
   mariadb1:
     image: mariadb

--- a/display.go
+++ b/display.go
@@ -66,7 +66,9 @@ func display() {
 	if !daemon {
 		termbox.Flush()
 		_, newlen := termbox.Size()
-		if newlen > termlength {
+		if newlen == 0 {
+			// pass
+		} else if newlen > termlength {
 			termlength = newlen
 			tlog.Len = termlength - 9 - (len(hostList) * 3)
 			tlog.Extend()

--- a/repmgr.go
+++ b/repmgr.go
@@ -227,6 +227,9 @@ var monitorCmd = &cobra.Command{
 			termlength = 40
 		} else {
 			_, termlength = termbox.Size()
+			if termlength == 0 {
+				termlength = 120;
+			}
 		}
 		loglen := termlength - 9 - (len(hostList) * 3)
 		tlog = termlog.NewTermLog(loglen)

